tst/testinstall.g

```diff
@@ -70,6 +70,7 @@ RunStandardTests( [
   [ "oprt.tst", 2000000 ],
   [ "ratfun.tst", 800000 ],
   [ "relation.tst", 7700000 ],
+  [ "recordname.tst", 100000 ],
```

Comment:
Just to note that we used to have the setting when the list of files listed in `testinstall.g` is a subset of `testall.g`, but I am happy for this requirement to be abandoned. Will merge this PR now.